.github/workflows/static-code-analysis.yml

```diff
@@ -0,0 +1,13 @@
+name: Static code analysis
+
+on: [push]
+
+jobs:
+  static-code-analysis:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run PHPStan
+        uses: docker://jakzal/phpqa:php7.3-alpine
```

Comment:
I've never seen this action. 
I've always used https://github.com/OskarStark/phpstan-ga

Are they any difference?

Reply:
The **@jakzal / phpqa** image actually isn't Github Actions nor phpstan specific, so allows execution of dozens of qa tools and multiple versions of php (and an Alpine base). See https://github.com/jakzal/phpqa for more info

Does OskarStark/phpstan-ga add extra integration features with Github Actions?

Reply:
Oh. I see. Whatever works :)

Oskar’s tool is just phpstan